lib/contact_screen.dart

```diff
@@ -0,0 +1,317 @@
+                  ? contactsElement.phones[0].number
+                  : ""));
+        }
+      });
+      _searchNotifier.updateSearchValue(_contactsList);
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      floatingActionButton: FloatingActionButton.extended(
+        onPressed: () {
+          Navigator.pop(context, _selectedContactsList);
+        },
+        label: const Text('Done'),
+        icon: const Icon(Icons.check, color: Colors.green),
+        backgroundColor: Colors.pink,
+      ),
+      appBar: PreferredSize(
+        preferredSize: const Size(double.infinity, kToolbarHeight),
+        child: ValueListenableBuilder(
+          valueListenable: _valueNotifier,
+          builder: (context, value, child) {
+            return _valueNotifier.value
+                ? appBarForShowOnlyTitle()
+                : appBarForSearch();
+          },
+        ),
+      ),
+      body: _isLoading
+          ? const Center(child: CircularProgressIndicator())
+          : ValueListenableBuilder<List<ContactModel>>(
+              valueListenable: _searchNotifier.contactNotifier,
+              builder: (context, filterList, child) {
+                return ListView.builder(
+                  itemCount: filterList.length,
+                  shrinkWrap: true,
+                  itemBuilder: (BuildContext context, int index) {
+                    return InkWell(
+                      onTap: () {
+                        if (widget.contactSelection ==
+                            ContactSelection.singleSelect) {
+                          Navigator.pop(context, filterList[index]);
+                        } else {
+                          final localContact =
+                              _selectedContactsList.firstWhereOrNull((element) =>
+                                  element.name == filterList[index].name);
+                          if (localContact == null) {
+                            _selectedContactsList.add(filterList[index]);
+                          } else {
+                            _selectedContactsList.remove(localContact);
+                          }
+                          setState(() {});
+                        }
+                      },
+                      child: Column(
+                        children: [
+                          (index > 0 &&
+                                  filterList[index].kilvishId == null &&
+                                  filterList[index].kilvishId != null)
+                              ? const Divider(height: 1, color: Colors.grey)
+                              : const SizedBox(),
+                          ListTile(
+                            leading: Stack(
+                              children: [
+                                CircleAvatar(
+                                  child: Text(filterList[index].name.isNotEmpty
+                                      ? filterList[index].name[0]
```

Comment:
should this not be `filterList[index].name` .. is name an array ?

Reply:
We need only first character to show in circle
![image](https://github.com/tamraj-kilvish/kilvish/assets/28701585/f785d84d-74bb-40f3-b39c-2a383a2badeb)